File: java/facadePattern/Statistics.java
package facadePattern;

public class Statistics {
	
	private DataFacade facade;
	private ClientData client;

	public Statistics(DataFacade facade, ClientData client) {
		this.facade = facade;
		this.client = client;
	}

	public void getData(String id) {
		String[] allData = this.facade.retrieveData(id);
		this.client.setPassengerDetails(allData);
		this.client.setFlightDetails(allData);
		this.client.setBaggageDetails(allData);
	}
}

File: php/facadePattern/Statistics.php
<?php namespace php\facadePattern;

class Statistics{

	private $facade;

	public function __construct(DataFacade $facade){ 
		$this->facade = $facade;
	}

	public function retrieveData($passID){
		$clientData = new ClientData();
		$allData = $this->facade->retrieveStats($passID);
		$clientData->setPassenger($allData['passenger']);
		$clientData->setFlight($allData['flight']);
		$clientData->setBaggage($allData['baggage']);
	}
}